deeppavlov/models/embedders/fasttext_embedder.py

```diff
@@ -80,27 +83,28 @@ def load(self, *args, **kwargs):
                 raise ConfigError("Provided `load_path` for {} doesn't exist!".format(
                     self.__class__.__name__))
         else:
-            warn("No `load_path` is provided for {}".format(self.__class__.__name__))
-            if self.embedding_url:
-                try:
-                    print('[trying to download a pretrained fasttext model from repository]')
-                    local_filename, _ = urllib.request.urlretrieve(self.embedding_url)
-                    with open(local_filename, 'rb') as fin:
-                        model_file = fin.read()
-
-                    mp = self.save_path
-                    self.load_path = self.save_path
-                    model = self.load()
-                    print("[saving downloaded fasttext model to {}]".format(mp))
-                    with open(str(mp), 'wb') as fout:
-                        fout.write(model_file)
-                except Exception as e:
-                    raise RuntimeError(
-                        'Looks like the provided fasttext url is incorrect', e)
-            else:
-                raise FileNotFoundError(
-                    'No pretrained fasttext model provided or provided "load_path" is incorrect.'
-                    ' Please include "load_path" to json.')
+            try:
+                log.warning("No `load_path` is provided for {}".format(self.__class__.__name__))
+                if self.embedding_url:
+                    try:
+                        log.info('[trying to download a pretrained fasttext model from repository]')
+                        local_filename, _ = urllib.request.urlretrieve(self.embedding_url)
+                        with open(local_filename, 'rb') as fin:
+                            model_file = fin.read()
+
+                        mp = self.save_path
+                        self.load_path = self.save_path
+                        model = self.load()
+                        log.info("[saving downloaded fasttext model to {}]".format(mp))
+                        with open(str(mp), 'wb') as fout:
+                            fout.write(model_file)
+                    except Exception as e:
+                        log.error('Looks like the provided fasttext url is incorrect', exc_info=True)
+                else:
+                    raise FileNotFoundError
+            except Exception:
+                log.error('No pretrained fasttext model provided or provided "load_path" is incorrect.'
+                          ' Please include "load_path" to json.', exc_info=True)
```

Comment:
Do we not need to `raise` here after logging the error?

Reply:
Fixed with `sys.exit()`